hbase-common/src/main/java/org/apache/hadoop/hbase/HConstants.java

```diff
@@ -1445,6 +1445,15 @@
       "hbase.util.default.lossycounting.errorrate";
   public static final String NOT_IMPLEMENTED = "Not implemented";
 
+  // Default TTL - FOREVER
+  public static final long DEFAULT_SNAPSHOT_TTL = 0;
+
+  // User defined Default TTL config key
+  public static final String DEFAULT_SNAPSHOT_TTL_CONFIG_KEY = "hbase.master.snapshot.ttl.default";
```

Comment:
This should be "hbase.master.snapshot.ttl" because you are setting an effective value. 

(Calling the default constant DEFAULT_SNAPSHOT_TTL is fine because that actually is naming a default value. )

Minor point. If you like.